static/app/utils/profiling/spanChart.spec.tsx

```diff
@@ -268,4 +268,68 @@ describe('spanChart', () => {
     expect(chart.configSpace.height).toBe(1);
     expect(chart.configSpace.width).toBe(10 * 1e9);
   });
+
+  it('does not infinitely loop if a span is truly orphaned', () => {
+    const start = Date.now();
+    const tree = new SpanTree(
+      txn({
+        contexts: {trace: {span_id: 'root'}},
+        startTimestamp: start + 0,
+        endTimestamp: start + 10,
+      }),
+      [
+        s({
+          span_id: '1',
+          parent_span_id: 'root',
+          timestamp: start + 10,
+          start_timestamp: start,
+        }),
+        s({
+          span_id: '2',
+          parent_span_id: undefined,
+          timestamp: start + 10,
+          start_timestamp: start,
+        }),
```

Comment:
Can we leave a comment somewhere about this? I think eventually we'll want to render these spans as children of the root span but for now this is fine.